libraries/AP_BattMonitor/AP_BattMonitor.cpp

```diff
@@ -91,9 +91,11 @@ AP_BattMonitor::init()
 
     convert_params();
 
-#if CONFIG_HAL_BOARD_SUBTYPE == HAL_BOARD_SUBTYPE_LINUX_BEBOP || CONFIG_HAL_BOARD_SUBTYPE == HAL_BOARD_SUBTYPE_LINUX_DISCO
-    // force monitor for bebop
-    _params[0]._type.set(AP_BattMonitor_Params::BattMonitor_TYPE_BEBOP);
+#ifdef BATT_MONITOR_DEFAULT
```

Comment:
Can we do HAL_BATT_MONITOR_DEFAULT? Just for name consistency.

Reply:
done